packages/components/src/internal/components/editable/utils.ts

```diff
@@ -153,7 +153,8 @@ export function getUpdatedDataFromGrid(
                 // updated values. This is not the final type check.
                 if (typeof originalValue === 'number' || typeof originalValue === 'boolean') {
                     try {
-                        value = JSON.parse(value);
+                        if (!isQuotedWithDelimiters(value, ','))
+                            value = JSON.parse(value);
```

Comment:
Just to be clear, when we populate the grid, we supply the needed quotes around the names, but if someone pastes values into the grid, we will require that they add the quotes, right?

Reply:
No, in that case we call `quoteValueWithDelimiters` on those values at `getUpdatedLineageRows`

Reply:
Yeah, ok. Thanks for the reminder.